packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/scheduler.controller.js

```diff
@@ -70,8 +70,12 @@ export default /* @ngInject */ function TelephonySchedulerCtrl(
 
     return self.scheduler
       .getEvents({
-        'dateStart.from': start.format(),
-        'dateEnd.to': end.format(),
+        'dateStart.to': moment(end)
+          .startOf('month')
```

Comment:
why ```startOf('month')```?

if should just be:
```
'dateStart.to': end.format(),
'dateEnd.from': start.format()
```
no ?

Reply:
what works:
- dateEnd.from = 2021-02-01 and dateStart.to = 2021-02-28 shows my event from 2021-01-29 to 2021-02-07
- dateEnd.from = 2020-12-28 and dateStart.to = 2021-01-31 shows my event from 2021-01-29 to 2021-02-07 too

Reply:
@julieni yes the `startOf()` was a remnant of previous code, it is removed.
Now with these parameters, you should be able to retrieve either:

- Events starting before and finishing after current month.
- All events which overlap start/end of the month
- classic ones

Reply:
yep last commit seems OK to me :+1: